src/components/api.js

```diff
@@ -0,0 +1,223 @@
+export default class Api {
+  constructor(options) {}
+
+  //methods
+  getInitialCards() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
+      headers: {
+        authorization: "4792ec92-cf1c-45a6-8740-0f5d63585faa",
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://snipboard.io/pRdaAZ.jpg . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)